projects/admin/src/app/acquisition/components/account/account-list/account-list.component.ts

```diff
@@ -31,30 +34,65 @@ export class AccountListComponent implements OnInit {
   /** Root account to display */
   rootAccounts: IAcqAccount[] = [];
 
+  /**
+   * Export options configuration.
+   */
```

Comment:
please use a single line comment to describe a component attribute.

Reply:
done